spring-boot-project/spring-boot/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -203,6 +203,18 @@
       "sourceType": "org.springframework.boot.context.logging.LoggingApplicationListener",
       "defaultValue": true
     },
+    {
+      "name": "logging.threshold.console",
+      "type": "java.lang.String",
+      "description": "Log level threshold for console output.",
+      "defaultValue": "TRACE"
```

Comment:
Isn't the default `DEBUG` rather than `TRACE`?

Reply:
It's filter level of appender, not log level of logger, TRACE means appender won't drop any log from logger by default.